### PR TITLE
Implement YAML Linting

### PR DESCRIPTION
**Pull request by apmody**
Resolves issue #1382

### CODE REVIEW

File: ontology/ddpheno.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ contact:
 license:
   url: http://creativecommons.org/publicdomain/zero/1.0/
   label: CC-0
-tracker: https://github.com/obophenotype/dicty-phenotype-ontology/issues
```
Comment (matentzn) [inline ontology/ddpheno.md:11]: shouldnt be removed

File: ontology/cteno.md [hunk 2]
```diff
@@ -20,15 +20,12 @@ homepage: https://github.com/obophenotype/ctenophore-ontology
...
 dependencies:
- - id: uberon
- - id: ro
+  - id: uberon
+  - id: ro
 taxon:
   id: NCBITaxon:10197
   label: Ctenophore
 tracker: https://github.com/obophenotype/ctenophore-ontology/issues
-jobs:
```
Comment (matentzn) [inline ontology/cteno.md:29]: why is this removed?

File: ontology/emapa.md [hunk 3]
```diff
@@ -19,19 +19,17 @@ taxon:
   id: NCBITaxon:10088
   label: Mus
 title: Mouse Developmental Anatomy Ontology
-tracker: https://github.com/obophenotype/mouse-anatomy-ontology/issues
```
Comment (matentzn) [inline ontology/emapa.md:22]: Why removed?

**Review by matentzn — CHANGES_REQUESTED** [review]
First of all, this is GREAT we do that, very nice :) This is an incomplete review:

I saw a bunch of tracker entries removed (see comments) and job entries; why?

**Comment by jamesaoverton** [comment]
@matentzn One of the things we're checking for is duplicate keys. I think that in all the cases you point out, there was another `tracker` or `job` with the same information.

I'm also reviewing to make sure we don't drop anything.

File: ontology/fbbt.md [hunk 4]
```diff
@@ -51,6 +51,8 @@ usages:
     example:
       - url: http://www.virtualflybrain.org/site/stacks/index.htm?id=FBbt_00003651
         description: "Ring neuron R2 in VFB"
+  - user: http://www.virtualflybrain.org/
+    description: VFB uses FBbt to annotate brain images
```
Comment (jamesaoverton) [inline ontology/fbbt.md:55]: I think this should be multiple examples instead of multiple users.

**Review by jamesaoverton — CHANGES_REQUESTED** [review]
There are two things I want to change, but I'll make those changes myself.

**Review by jamesaoverton — APPROVED** [review]
I went through this quite carefully, and I don't think anything has been lost.

The major benefit is that we've resolved many problems with duplicate keys. We've also fixed indentation, which can be a problem for interpreting YAML. A large number of changes are just removing leading and trailing whitespace, but I think that sort of clean up is also valuable to downstream consumers.

**Review by matentzn — APPROVED** [review]

**Comment by matentzn** [comment]
This is great! Thanks for the clarification of the "lost keys" :) approved now!
